src/promptflow/promptflow/_sdk/schemas/_run.py

```diff
@@ -51,7 +51,7 @@ class RunSchema(YamlFileSchema):
     )
     connections = fields.Dict(keys=fields.Str(), values=fields.Dict(keys=fields.Str()))
     # inputs field
-    data = LocalPathField()
+    data = fields.Str()
```

Comment:
We should have doc on the data we supportred in reference doc

Reply:
Updated in public func & class.